Tests/TestingApps/PurchaseTesterSwiftUI/Shared/ReceiptInspector.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  ContentView.swift
+//  ReceiptParser
+//
+//  Created by Andrés Boedo on 1/25/23.
+//
+
+import SwiftUI
+import ReceiptParser
+
+struct ReceiptInspectorView: View {
+    @State private var encodedReceipt: String = ""
+    @State private var parsedReceipt: String = ""
+    @State private var verifyReceiptResult: String = ""
+    @State private var sharedSecret: String = ""
+
+
+    var body: some View {
+        VStack {
+            Text("Receipt Parser")
+                .font(.title)
+                .padding()
+
+            TextField("Enter receipt text here (base64 encoded)", text: $encodedReceipt, onEditingChanged: { isEditing in
```

Comment:
main UI, with text field for receipt text, shared secret and the text fields for the receipt